docs/rules/padding-around-test-blocks.md

```diff
@@ -0,0 +1,39 @@
+# padding-around-test-blocks
+
+## Rule Details
+
+This rule enforces a line of padding before _and_ after 1 or more `test`/`it` statements
+
+Note that it doesn't add/enforce a padding line if it's the last statement in its scope
+
+Examples of **incorrect** code for this rule:
+
+```js
```

Comment:
These examples appear to be identical

Reply:
I did one set with `test` and one with `it`, just to show that both are supported. But lemme know if I'm misunderstanding the comment!

Reply:
ahhhhh. Just me being an idiot

Thanks!

Reply:
Nah. Your mind just knows it's an alias. You're too deep in the matrix.